multiversx_sdk/facades/entrypoints.py

```diff
@@ -0,0 +1,163 @@
+
```

Comment:
A few short unit tests would be nice. Can be based on the examples from the specs (but not necessarily):

https://github.com/multiversx/mx-sdk-specs/blob/main/facades/entrypoints.md#examples

Reply:
added a few.